tests/testthat/test-estimate_CUMINC.R

```diff
@@ -0,0 +1,76 @@
+test_that("estimate_cuminc() works", {
```

Comment:
Can you follow the testing conventions we use? You can have a look at our examples. Let me know if you need any help here.

Reply:
I updated trying to make it look the same. if it's not right, can you walk me through it or make the update?